webapp-django/crashstats/cron/management/commands/cronrun.py

```diff
@@ -60,18 +59,14 @@ def add_arguments(self, parser):
 
     def handle(self, **options):
         """Execute cronrun command."""
-        try:
-            if options['job']:
-                job_args = options.get('job_arg') or []
-                # Re-add the -- because they're optional arguments; note that this
-                # doesn't support positional arguments
-                cmd_args = ['--%s' % arg for arg in job_args]
-                return self.cmd_run_one(options['job'], options['force'], cmd_args)
-            else:
-                return self.cmd_run_all()
-        except Exception:
```

Comment:
I dropped this ``try`` block because the default handler does what the ``except`` block does - log the uncaught exception to Sentry on exit.